sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -242,7 +242,9 @@ class FieldText(FormContent):
         Units are in pixels for images and inches for PDF.
     :ivar text_content:
         When `include_text_content` is set to true, a list of text
-        elements constituting this field or value is returned.
+        elements constituting this field or value is returned. The list
+        constitutes of text elements such as lines and words in addition
+        to form fields.
```

Comment:
we shouldn't be getting FormFields back in text_content. what do you mean here?

Reply:
let me rephrase it